### PR TITLE
[#157875848] chore: add xml response

### PR DESCRIPTION
**Pull request by gunzip**
aside fixes package.json version to match npm package (incremented beforewards but wrongly not pushed)

### CODE REVIEW

**Comment by gunzip** [comment]
cc @lussoluca @cloudify

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  :tada: - congrats on your new release
  </td>
    </tr>
  </tbody>
</table>


## Affected stories

  * ⚙️ [#157875848](https://www.pivotaltracker.com/story/show/157875848): Aggiungere response XML e redirect a italia-ts-commons

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by lussoluca** [comment]
LGTM

File: src/responses.ts [hunk 1]
```diff
@@ -63,6 +64,54 @@ export function ResponseSuccessJson<T>(o: T): IResponseSuccessJson<T> {
...
+  readonly value: T; // needed to discriminate from other T subtypes
+}
+
+/**
+ * Returns a successful xml response.
+ *
+ * @param o The object to return to the client
+ */
+export function ResponseSuccessXml<T>(o: T): IResponseSuccessXml<T> {
+  const kindlessObject = Object.assign(Object.assign({}, o), {
```
Comment (cloudify) [inline src/responses.ts]: mmh when does `T`  have the `kind` attribute? `kind` is used to the response types, not for their contained `value`

Comment (gunzip) [reply src/responses.ts]: removed

File: src/url.ts [hunk 2]
```diff
@@ -1,23 +1,27 @@
 import * as t from "io-ts";
 import * as url from "url";
 
-const isUrl = (v: t.mixed): v is url.Url =>
-  v !== null && typeof v === "object" && "href" in v;
+export type ValidUrl = url.Url & { readonly href: string };
```
Comment (cloudify) [inline src/url.ts]: isn't `Url` supposed to describe a valid URL already? also `Url` already defines an [href attribute](https://nodejs.org/api/url.html#url_url_href)

Comment (gunzip) [reply src/url.ts]: nope. Url defines an optional href attribute which maybe undefined under some circumstances). 
we generally deal with http(s) url, so we need href to be always defined (that's the reason of this check). UrlFromString is a subset of all possible Url(s)

**Pull request merged**